deployments/helm/values.yaml

```diff
@@ -0,0 +1,157 @@
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+## Expose the beyla internal metrics service to be accessed from outside the cluster (LoadBalancer service).
+## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
+## ref: http://kubernetes.io/docs/user-guide/services/
+##
+service:
+  enabled: false
+  type: ClusterIP
+  clusterIP: ""
+  loadBalancerIP: ""
+  loadBalancerClass: ""
+  loadBalancerSourceRanges: []
+  port: 80
+  targetPort: '{{ include "beyla.internalMetricsPort" . }}'
+  ## Service annotations. Can be templated.
+  annotations: {}
+  labels: {}
+  portName: service
+  # Adds the appProtocol field to the service. This allows to work with istio protocol selection. Ex: "http" or "tcp"
+  appProtocol: ""
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+
+
+## See `kubectl explain deployment.spec.strategy` for more
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+updateStrategy:
+  type: RollingUpdate
+
+
+# Additional volumes on the output daemonset definition.
+volumes: []
+# - name: foo
+#   secret:
+#     secretName: mysecret
+#     optional: false
+
+# Additional volumeMounts on the output Deployment definition.
+volumeMounts: []
+# - name: foo
+#   mountPath: "/etc/foo"
+#   readOnly: true
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# To use an external configmap for Beyla - this will replace existing configmap from chart
+# kubectl create cm --from-file=beyla-config.yaml=<name-of-config-file> -n <namespace>
+# externalBeylaConfigMap: 
+
+
+## More configuration options available at https://grafana.com/docs/beyla/latest/configure/options/
+configmapData:
+  #   open_port: 8443
+  #   routes:
+  #     unmatched: heuristic
+  #   log_level: info
+  #   otel_traces_export:
+  #     endpoint: http://grafana-agent:4318
+  #  ## or alternatively use
+  #   # grafana:
+  #   #   otlp:
+  #   #     cloud_zone: prod-eu-west-0
+  #   #     cloud_instance_id: 123456  
+  #   #     cloud_api_key:  
+  #   discovery:
+  #     services:
+  #       - k8s_namespace: default
+  #   attributes:
+  #     kubernetes:
+  #       enable: true
+  # internal metrics reporting. Refer: https://grafana.com/docs/beyla/latest/configure/options/#internal-metrics-reporter
+  # If set, user can expose the metrics endpoint via k8s service by configuring .Values.service section
+  #prometheus_export:
+    #port: 9090
+    #path: /metrics
```

Comment:
@arramos84 
> At a minimum, it should work with the default values.

That's an interesting point. Beyla requires at least two input values: the service to instrument and the way to export the data (e.g. the OpenTelemetry endpoint or the Prometheus exporter pod).

So I guess that, to let Beyla work with some default inputs, we should provide something like:

```yaml
# look for ALL the services in the host
discovery:
  services:
    - k8s_namespace: .
# export metrics as prometheus metrics by default
prometheus_export:
  port: 9090
# enable kubernetes
attributes:
  kubernetes:
    enable: true
```

Could we set the above YAML as the default `configMapData`?